graviti/manager/draft.py

```diff
@@ -83,9 +83,17 @@ def __setitem__(self, key: str, value: DataFrame) -> None:
         if is_replace:
             self.operations.append(DeleteSheet(key))
 
+        if value.operations is None:
+            value.operations = [AddData(value.copy())]
+        else:
+            raise NotImplementedError(
+                "Not support assigning one DataFrame to multiple sheets."
+                " Please use method 'copy' first."
+            )
         self.operations.append(CreateSheet(key, value.schema.copy()))
 
     def __delitem__(self, key: str) -> None:
+        self[key].operations = None
         super().__delitem__(key)
```

Comment:
`super().pop(key)`?

Reply:
but `pop` will call `__delitem__` recursively

Reply:
will `super().pop()` call `Draft.__delitem__`?

Reply:
Yes, I tried and failed.